### ISSUE TITLE
[Feature Request] History to see what you've read

### ISSUE THREAD

**Issue by ShockedCoder** [AriaMoradi/Tachidesk#453]
## What feature should be added to Tachidesk?
When you read/start a manga or smth it gets saved to your history so that you don't lose it

##  Why/Project's Benefit/Existing Problem
You don't need to go searching for that one manga you accidentally clicked off of, or maybe even your browser suddenly crashed and now you've lost it, but with this, you can just easily continue where you left off without worrying about losing it

**Comment by hitokirihahn** [comment]
Yes, i really like this feature from tachiyomi. Please add this feature.

**Comment by Syer10** [comment]
Marking as completed since most UIs now use the Chapter.lastReadAt property to show history.
